### PR TITLE
Change addon exported files from .coffee to .js

### LINKED ISSUE #131: Addons created in coffeescript can only be consumed by applications with coffeescript
I discovered that components created using coffeescript in an addon can not be consumed by an application that is not set up with `ember-cli-coffeescript`. This seems to be caused by the wrapping files under `app` being generated as coffee-files which the consuming application does not know how to handle. My suggestion would be to create these proxy-files as pure javascript files so that any addons created using coffeescript can be used by all applications regardless of language preferences. 

What do you think @kimroen? I can create a PR for that, if you agree on the resolution.

### PR DESCRIPTION
**Pull request by mriska**
This PR changes the `*-addon` blueprints to use javascript files instead of coffeescript files. Without this change, addons using `ember-cli-coffeecript` creates files that can't be consumed in a application not using `ember-cli-coffeescript`.

Fixes #131

### CODE REVIEW

**Comment by mriska** [comment]
Same here, will fix to make tests pass.

**Comment by mriska** [comment]
@kimroen, pushed updated code with amended tests. Travis builds green now.

**Comment by mriska** [comment]
@kimroen, What do you think of this? Is it merge-worthy?

File: node-tests/blueprints/addon-import-test.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ describe('Acceptance: ember generate and destroy addon-import', function() {
...
     return emberNew()
       .then(() => emberGenerateDestroy(args, (file) => {
-        var importFile = file('app/addon-imports/foo.coffee');
+        var importFile = file('app/addon-imports/foo.js');
 
         expect(importFile)
-          .to.contain("export { default } from 'my-app/addon-imports/foo'");
+          .to.contain("export { default } from 'my-app/addon-imports/foo';");
 
         expectCoffee(importFile);
```
Comment (kimroen) [inline node-tests/blueprints/addon-import-test.js]: These lines expect that the contents of the file in question is valid CoffeeScript, and that is not relevant for these tests that only involve JavaScript, so it should be removed.

**Comment by mriska** [comment]
@kimroen the incorrect `expectCoffee` is now removed from the tests. I would suggest merging this now - I have created a new issue #140 for investigating if the `addon` blueprints can be removed completely. I'll be happy to take a look at that one separately.

**Comment by kimroen** [comment]
Great, thank you again!

**Comment by perlun** [comment]
Could we get a v1.16.1 with this fix included @kimroen? Pretty please with sugar on top. 😇

**Comment by kimroen** [comment]
> Could we get a v1.16.1 with this fix included @kimroen? Pretty please with sugar on top. 😇

You absolutely could, but I'm in the middle of moving so I'm short on time. Hopefully soon!

**Comment by kimroen** [comment]
There we go! [`v1.16.1`](https://github.com/kimroen/ember-cli-coffeescript/releases/tag/v1.16.1)

**Pull request merged**